apps/server/src/shared/infra/antivirus/antivirus.module.ts

```diff
@@ -24,7 +27,22 @@ export class AntivirusModule {
 						routingKey: options.routingKey,
 					},
 				},
+				{
+					provide: NodeClam,
+
+					useFactory: () =>
+						new NodeClam().init({
+							debugMode: true,
```

Comment:
Maybe we can disable the `debugMode` or is it still required?

Reply:
Just for test case, i remove it before merge